test/cpp/jit/test_gpu.cpp

```diff
@@ -2083,6 +2083,151 @@ void testGPU_FusionComputeAtNoCommonConsumer() {
   TORCH_CHECK(at::allclose(kernel_tv6, t6));
 }
 
+void checkConcretized(
```

Comment:
it's not critical for tests, but this kind of local helpers can be placed in anonymous namespaces

Reply:
Updated, thanks!